src/sdk/bucketspace_fs.js

```diff
@@ -130,13 +146,23 @@ class BucketSpaceFS extends BucketSpaceSimpleFS {
         try {
             const bucket_config_path = this._get_bucket_config_path(name);
             dbg.log0('BucketSpaceFS.read_bucket_sdk_info: bucket_config_path', bucket_config_path);
-            const { data } = await nb_native().fs.readFile(this.fs_context, bucket_config_path);
+            let { data } = await nb_native().fs.readFile(this.fs_context, bucket_config_path);
             const bucket = JSON.parse(data.toString());
             nsfs_schema_utils.validate_bucket_schema(bucket);
-            const is_valid = await this.check_bucket_config(bucket);
+            let is_valid = await this.check_bucket_config(bucket);
             if (!is_valid) {
                 dbg.warn('BucketSpaceFS: one or more bucket config check is failed for bucket : ', name);
             }
+            //also get bucket's account
+            const account_config_path = this._get_account_config_path(bucket.owner_account);
+            data = (await nb_native().fs.readFile(this.fs_context, account_config_path)).data;
+            const account = JSON.parse(data.toString());
+            nsfs_schema_utils.validate_account_schema(account);
+            is_valid = await this.check_bucket_config(bucket);
+            if (!is_valid) {
+                dbg.warn('BucketSpaceFS: account linked to bucket is not valid: ', name);
```

Comment:
should we throw an error here and not skip silently? Also, the warning does not match the issue.

Reply:
This check is a mistake, I've removed it.